src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/NotificationToDefendantService.java

```diff
@@ -48,6 +51,25 @@ public void notifyDefendant(Claim claim) {
         );
     }
 
+    public void notifyDefendantOfRejection(Claim claim) {
+        Response response = claim.getResponse().orElseThrow(IllegalArgumentException::new);
+
+        if (isResponseStatesPaid(response) || response.getResponseType() == ResponseType.PART_ADMISSION) {
```

Comment:
You are adding two level of filtering to choose ` notifyDefendant(claim);`  once in EventListener `sendNotificationToDefendant(ClaimantResponseEvent event)` and then here, Please move this condition in event listener if at all needed. 

This condition might be un-necessary as well, please verify.

Reply:
I agree
AC states rejection of part admission or states paid. Please note even full defence states paid less than claim amount is sent to claim-store as part-admission states paid hence all of the logic can be put at one place.